llm-agent-tests/gemini/test_gemini_env.py

```
@@ -0,0 +1,73 @@
+import os
+import sys
```

Ruff (0.11.9)

`sys` imported but unused

Remove unused import: `sys`

(F401)